ecommerce-ui/src/BnbDetails/BnbDetails.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import './BnbDetails.css';
+import airbnbs from '../airbnbs.json';
+import SuperHostIcon from '../HostIcon/SuperHostIcon';
+import HostIcon from '../HostIcon/HostIcon';
+import BookForm from './BookForm';
+
+class BnbDetails extends Component {
+
+  getHostIcon() {
+    return (
+      <div className="host-icon">
+        <HostIcon />
+      </div>
+    );
+  }
+
+  getSuperHostIcon() {
+    return (
+      <div className="host-icon">
+        <SuperHostIcon />
+      </div>
+    );
+  }
+
```

Comment:
I had to repeat these two functions in BnbDetails and BnbListings. I need to figure out a way to put them in one place and then call them in the two locations.

Reply:
I'd have one component covering both HostIcon and SuperHostIcon, and it would have a prop of isSuperHost.   Then on line 54 you could render that component directly.

Reply:
I need to experiment with this more. I couldn't get it to work because I couldn't figure out how to import functions from sibling components and then use them in conditional statements. Specifically how to rewrite line 54:

{isSuperHost ? this.getSuperHostIcon() : this.getHostIcon() }

What would "This" be? the imported component name?

But there's got to be a way. One component seems logical I'll just have to mess around with it some more.